### PR TITLE
docs: add create custom theme tutorial

### PR DESCRIPTION
**Pull request by dtanp-rft**
Add tutorial page for creating custom theme.

### CODE REVIEW

File: documents/src/pages/build-app/custom-theme.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+@button-pressed-background-color: @scheme-color-primary;
+```
+
+List of all less variables that you could override are in [Elemental Theme](https://github.com/Refinitiv/refinitiv-ui/blob/v6/packages/elemental-theme/src/variables.less) and [Halo Theme](https://github.com/Refinitiv/refinitiv-ui/blob/v6/packages/halo-theme/src/variants/dark/variables.less).
+
+@> Elemental Theme is super base theme of every themes in Element Framework, including Halo Theme. With the same concept as creating custom theme, Halo Theme is created by extending from Elemental Theme and overriding less variables, [see example](https://github.com/Refinitiv/refinitiv-ui/blob/v6/packages/halo-theme/src/variants/dark/overrides.less).
+
+## Customise theme native styles
+
+Some styles are required to be applied in native HTML elements such as font-family. Those styles can be set in less files in `native-elements` folder. For example, in the custom theme sample project, font-family and padding are customised for body HTML tag.
```
Comment (wsuwt) [inline documents/src/pages/build-app/custom-theme.md]:
```suggestion
Some styles are required to be applied into native HTML elements such as `font-family` to `body`. To so, you can add less file with name same as HTML tag to `native-elements` folder.

For example, in the custom theme sample project, `font-family` and `padding` are customised for body HTML tag.
```

File: documents/src/pages/build-app/custom-theme.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+// Customise native component style
+body {
+  padding: 20px;
+  font-family: sans-serif;
+}
+```
+
+## Customise component styles
+
+When styles of components are needed to be customised, you can customise styles of an individual Element Framework component by adding less file(s) in `custom-elements` folder. Custom theme sample project shows how you can customise styles of host and an internal part of `ef-toggle`.
```
Comment (wsuwt) [inline documents/src/pages/build-app/custom-theme.md]:
```suggestion
To customise a component styles, add `less` file with the same name as component's name into `custom-elements` folder. 

Provided sample project shows how you can customise styles of host and an internal part of `ef-toggle`.
```

File: documents/src/pages/build-app/custom-theme.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+  [part="toggle"] {
+    &::after {
+      border-radius: 50%;
+    }
+  }
+}
+```
+
+You only need to create less file for components that you need to customise. Any components that do not need any customisation, theme compiler will automatically extends it directly from Halo Theme and output the files in `./dark` and `./light` folder.
```
Comment (wsuwt) [inline documents/src/pages/build-app/custom-theme.md]:
```suggestion
You only need to create less file for components that you need to customise. The rest of components that do not need further customisation will be automatically extended from Halo theme.
```

File: documents/src/pages/build-app/custom-theme.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+```less
+@scheme-color-primary: @color-seafoam-400;
+@scheme-color-tertiary: @color-gray-600;
+@button-hover-background-color: @scheme-color-primary;
+@button-pressed-background-color: @scheme-color-primary;
+```
+
+List of all less variables that you could override are in [Elemental Theme](https://github.com/Refinitiv/refinitiv-ui/blob/v6/packages/elemental-theme/src/variables.less) and [Halo Theme](https://github.com/Refinitiv/refinitiv-ui/blob/v6/packages/halo-theme/src/variants/dark/variables.less).
+
+@> Elemental Theme is super base theme of every themes in Element Framework, including Halo Theme. With the same concept as creating custom theme, Halo Theme is created by extending from Elemental Theme and overriding less variables, [see example](https://github.com/Refinitiv/refinitiv-ui/blob/v6/packages/halo-theme/src/variants/dark/overrides.less).
```
Comment (wsuwt) [inline documents/src/pages/build-app/custom-theme.md]:
```suggestion
@> Elemental theme is a base theme of every themes in Element Framework, including Halo theme. Similarly, the same concept applies when creating custom theme, Halo theme is created by extending from Elemental theme and overriding less variables, [see example](https://github.com/Refinitiv/refinitiv-ui/blob/v6/packages/halo-theme/src/variants/dark/overrides.less).
```

**Review by wsuwt — DISMISSED** [review]

**Review by Nantawat-Poothong — DISMISSED** [review]

File: documents/src/navigation.md [hunk 5]
```diff
@@ -89,6 +89,7 @@ type: nav
     - [Overview](./accessibility/overview)
   - [Internationalization (i18n)](./intl/internationalization)
   - [Theme Switching](./guides/theme-switching)
+  -
```
Comment (wattachai-lseg) [inline documents/src/navigation.md]: Do we need this empty list item?

Comment (dtanp-rft) [reply documents/src/navigation.md]: Well spot! Thanks and fixed.

**Review by Nantawat-Poothong — APPROVED** [review]

**Pull request merged**